membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -297,13 +311,96 @@ def test_analysis_mean(self, universe):
         avg_mean = mc.results.average_mean
         assert_almost_equal(avg_mean, expected_mean)
 
-
-    def test_analysis_mean_default_selection(self, universe):
+    # test using wrap=False
+    def test_analysis_mean_no_wrap(self, universe):
         expected_mean = np.array([[7.50000000e+00,  1.33985392e-01,  2.77315457e-04],
                                   [-2.77315457e-04, -3.53944270e-01, -7.50000000e+00],
                                   [-2.77315457e-04, -5.01100068e-01, -7.50000000e+00]])
         mc = MembraneCurvature(universe,
+                               select='name PO4',
+                               n_x_bins=3,
+                               n_y_bins=3,
+                               wrap='False').run()
```

Comment:
```suggestion
                               wrap=False).run()
```

This string will still evaluate truthily, i.e. wrapping is still turned on. So, it's hard for this test to evaluate whether `wrap=False` is working. Switching to a new universe (maybe with dummy coordinates if that's easiest) where `wrap=True` and `wrap=False` give you different results might be the best way?

Reply:
Sorry about that and thanks for spotting it.